garak/generators/nim.py

```diff
@@ -42,6 +42,9 @@ class NVOpenAIChat(OpenAICompatible):
     active = True
     supports_multiple_generations = False
     generator_family_name = "NIM"
+    frequency_penalty = None
+    presence_penalty = None
```

Comment:
Based on this PR, I suspect these values really should be `instance` instead of `class` level, meaning these should be added to `DEFAULT_PARAMS` above instead of here.